src/radiant.c

```diff
@@ -2948,11 +2986,18 @@ int radiant_set_delay_settings(radiant_dev_t * bd, uint8_t rf0_delay, uint8_t rf
     return -1;
   }
 
+  //write physical values
   radiant_set_mem(bd, DEST_FPGA,RAD_REG_LAB_CTRL_RF0_DELAY,1,(uint8_t*)&rf0_delay);
   radiant_set_mem(bd, DEST_FPGA,RAD_REG_LAB_CTRL_RF0_DELAY_MASK,1,(uint8_t*)&rf0_delay_mask);
   radiant_set_mem(bd, DEST_FPGA,RAD_REG_LAB_CTRL_RF1_DELAY,1,(uint8_t*)&rf1_delay);
   radiant_set_mem(bd, DEST_FPGA,RAD_REG_LAB_CTRL_RF1_DELAY_MASK,1,(uint8_t*)&rf1_delay_mask);
 
```

Comment:
you may want to check return values here to make sure you're consistent (or, even just read them back).